### PR TITLE
Nitpicks and small changes - proposal

### PR DESCRIPTION
**Pull request by jubeira**
Proposing a few rewordings and reorderings.

### CODE REVIEW

File: full_integration_tutorial.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
-This tutorial intends to give a step-by-step guide into how to use ROS tools in conjunction with an Ignition based simulation. For this tutorial, we will be using Ignition Dome and ROS Melodic, however, please note that this tutorial is valid for Ignition releases compatibles with `ros_ign_bridge` and ROS releases compatibles with `teleop_twist_keyboard` and `ros_ign_bridge`.
+This tutorial intends to give a step-by-step guide into how to use ROS tools in conjunction with an Ignition based simulation. 
+You should be able to achieve a result like the one in this [video](https://youtu.be/O2LIB4F2XVQ) at the end of the tutorial.
 
-First will be presented the steps to complete the tutorial, and later will be explained what's going on behind the scene.
+We will be using Ignition Dome and ROS Melodic; however, please note that this tutorial is valid for Ignition releases that are compatible with `ros_ign_bridge` and ROS releases that are compatible with `teleop_twist_keyboard` and `ros_ign_bridge`.
+
+We'll start with the steps to complete the tutorial, and later will be explained what's going on behind the scenes.
 
 ## Pre-Requisites
```
Comment (jubeira) [inline full_integration_tutorial.md:10]: Is there a ROS installation that already contains ignition dome and the remaining dependencies?

I would try to narrow down these steps a bit.
Some options: 
- Bundle the tutorial as a small ROS package. Declare `ros_ign_bridge` and the teleop as dependencies in the package file, and some launchfiles as part of this repo. The install / run instructions can then be narrowed down to standard steps:
  - Clone repo inside a catkin workspace
  - Run rosdep to get ignition bridge and the teleop if necessary
  - catkin_make + source devel/setup.bash + launchfile
- Keep this list as is, but replace external links with single commands. E.g. you can probably get ignition bridge and teleop with `apt get`. For Ignition Dome and ROS, it's ok to have external links.

**Review by jubeira — COMMENTED** [review]
One more overall comment: I'm not sure for Ignition Gazebo, but the run instructions can probably be shortened with a small launchfile that runs everything (except maybe the teleop, we'd need to check that one).

I've left one more comment below regarding the installation. While it looks fine as is, if the target audience is beginners to ROS / Ignition, the more we polish them, the better.

**Review by Blast545 — APPROVED** [review]

File: full_integration_tutorial.md (hunk 1 above)
Comment (Blast545) [reply full_integration_tutorial.md:10]: I added your comments on top of: 523bbd1. Didn't merge this directly as there were conflicting issues related to the file that I erased.

Comment (Blast545) [reply full_integration_tutorial.md:10]: Also replaced install instructions links where instructions where short

**Pull request closed without merging**